pom.xml

```diff
@@ -100,8 +100,11 @@
         <module>lib/trino-plugin-toolkit</module>
         <module>lib/trino-rcfile</module>
         <module>lib/trino-record-decoder</module>
+
```

Comment:
Just to prevent potential merge conflicts in the future, can we eliminate the extra whitespace being added in this file? Lines 103, 104 and 107.